css/style.css

```diff
@@ -1,7 +1,7 @@
-.form-wrapper{
+.form-wrapper {
```

Comment:
Can add more css to this project instead of just using bootstrap.

Reply:
- [ ] It would be better to make a polite comment instead of comments looking like an order. For example; this can be written as `It would be better if we add more CSS to this project instead of using the CSS framework (Bootstrap)`

Reply:
Thank you Daniel for the suggestion.

Reply:
- [ ] It would be better if we add why to our comments; for example; `why must students use more of CSS as compared to Bootstrap?`. Using more CSS helps us understand styling and make us write better code withour repetition